operators/admin/olm-adding-operators-to-cluster.adoc

```diff
@@ -20,10 +20,17 @@ If you have the pull secret, add the `redhat-operators` catalog to the `Operator
 endif::[]
 
 include::modules/olm-installing-operators-from-operatorhub.adoc[leveloffset=+1]
-include::modules/olm-installing-from-operatorhub-using-web-console.adoc[leveloffset=+2]
+* xref:../../operators/understanding/olm-understanding-operatorhub.adoc#olm-understanding-operatorhub[Understanding OperatorHub]
+
+include::modules/olm-installing-from-operatorhub-using-web-console.adoc[leveloffset=+1]
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
-include::modules/olm-installing-from-operatorhub-using-cli.adoc[leveloffset=+2]
+include::modules/olm-installing-from-operatorhub-using-cli.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../../operators/understanding/olm/olm-understanding-operatorgroups.adoc#olm-operatorgroups-about_olm-understanding-operatorgroups[About Operator groups]
+
+include::modules/olm-installing-specific-version-cli.adoc[leveloffset=+1]
+.Additional resources
```

Comment:
So we've updated the guidelines to use `==` for additional resources in assembly files: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#writing-assemblies

Reply:
This is an additional resource for the module, not the assembly.